index.html

```diff
@@ -36,6 +36,20 @@
       </div>
     </div>
 
+    <div id="call-for-proposals" class="sponsors">
+      <section>
+        <h2>Call for Proposals is Now Open</h2>
+
+        Rustfest has 8-10 regular speaking slots. Rust is a new language so all proposals are welcome. If you have been working on something for only a few months, but it is interesting, then apply!
```

Comment:
Yep, I like your suggestion much better, I just stole the text from a year ago! Will change it.